easybuild/easyconfigs/b/Bowtie2/Bowtie2-2.2.9-intel-2016b.eb

```diff
@@ -0,0 +1,48 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# Modified by: Robert Schmidt <rjeschmi@gmail.com>
+# Ottawa Hospital Research Institute - Bioinformatics Team
+# Modified by: Adam Huffman <adam.huffman@crick.ac.uk>
+# The Francis Crick Institute
+# Modified by: Kurt Lust, UAntwerp: Overwrite compiler options used in the GNU-based 
+# Makefile with the preferred options for intel compilers.
+
+easyblock = 'MakeCp'
+
+name = 'Bowtie2'
+version = '2.2.9'
+
+homepage = 'http://bowtie-bio.sourceforge.net/bowtie2/index.shtml'
+description = """ Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads 
+ to long reference sequences. It is particularly good at aligning reads of about 50 up to 100s or 1,000s 
+ of characters, and particularly good at aligning to relatively long (e.g. mammalian) genomes. 
+ Bowtie 2 indexes the genome with an FM Index to keep its memory footprint small: for the human genome, 
+ its memory footprint is typically around 3.2 GB. Bowtie 2 supports gapped, local, and paired-end alignment modes."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+sources = ['%(namelower)s-%(version)s-source.zip']
+source_urls = [('http://sourceforge.net/projects/bowtie-bio/files/%(namelower)s/%(version)s', 'download')]
+
+toolchainopts = {'cstd': 'c++11'}
+
+# Make sure that the Makefile uses options for intel rather than the ones for gcc coded
+# in the Makefile by setting RELEASE_FLAGS.
+buildopts = 'RELEASE_FLAGS="$CXXFLAGS"'
```

Comment:
We should also apply this same fix to the other Bowtie2 easyconfigs (in a separate PR).

While looking into that, I noticed that we have a Bowtie2 easyblock that is only used by 3/18 Bowtie2 easyconfigs, we should clean that up, probably by enhancing the Bowtie2 easyblock to specify `RELEASE_FLAGS`, and to make sure it copies everything that the easyconfigs now copy...

I'll try and find some time for that.